### PR TITLE
add inheritance-js w/ npm autoupdate

### PR DESCRIPTION
**Pull request by pvnr0082t**
cc #4751 
@kennynaoh @x09326 Please help me review this pull request, thank you.
Manually add v0.4.3 from git repo because npm auto-update can't get

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bsara/inheritance.js
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/inheritance-js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1
   - Count of watchers: 1
   - Count of forks: 2
 * NPM download stats (optional):
   - Downloads in the last day: 23
   - Downloads in the last week: 187
   - Downloads in the last month: 705

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #4751, cc @bsara

### CODE REVIEW

File: ajax/libs/inheritance-js/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "inheritance-js",
+  "filename": "inheritance.min.js",
+  "version": "0.4.12",
+  "description": "Simple, lightweight extensions and helpers that make inheritance in JS a breeze, all with pure JavaScript, no extra libraries needed",
+  "license": "CPOL-1.02",
+  "repository": {
+    "type": "git",
+    "url": "bsara/inheritance.js"
```
Comment (x09326) [inline ajax/libs/inheritance-js/package.json]: @pvnr0082t 
Is this url okay?
I think it can be more complete.

Comment (pvnr0082t) [inline ajax/libs/inheritance-js/package.json]: @x09326 @kennynaoh I have fixed it. Please help me review it again, thank you.

**Comment by kennynaoh** [comment]
LGTM

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Thanks!

**Pull request merged**